docs/_Interface_IColonyNetwork.md

```diff
@@ -0,0 +1,858 @@
+---
+title: IColonyNetwork
+section: Interface
+order: 2
+---
+
+  
+
+## Interface Methods
+
+
+
+### `addColonyVersion`
+
+Adds a new Colony contract version and the address of associated `_resolver` contract. Secured function to authorised members
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_version|uint256| The new Colony contract version
+|_resolver|address| Address of the `Resolver` contract which will be used with the underlying `EtherRouter` contract
+
+
+
+
+
+### `addr`
+
+Returns the address the supplied node resolves do, if we are the resolver
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|node|bytes32| The namehash of the ENS address being requested
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_undefined_|address|
+
+
+
+### `addSkill`
+
+Adds a new skill to the global or local skills tree, under skill `_parentSkillId`
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_parentSkillId|uint256| Id of the skill under which the new skill will be added. If 0, a global skill is added with no parent
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|skillId|uint256| Id of the added skill
+
+
+
+### `appendReputationUpdateLog`
+
+Adds a reputation update entry to log
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_user|address| The address of the user for the reputation update
+|_amount|int256| The amount of reputation change for the update, this can be a negative as well as a positive value
+|_skillId|uint256| The skill for the reputation update
+
+
+
+
+
+### `calculateMinerWeight`
+
+Calculate raw miner weight in WADs
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_timeStaked|uint256| Amount of time (in seconds) that the miner has staked their CLNY
+|_submissonIndex|uint256| Index of reputation hash submission (between 0 and 11)
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|minerWeight|uint256| The weight of miner reward
+
+
+
+### `createColony`
+
+Creates a new colony in the network
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_tokenAddress|address| Address of an ERC20 token to serve as the colony token
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_undefined_|address|
```

Comment:
Badly parsed return parameter

Reply:
This one neither matches the name (no name) or type (`address`)

```
  /// @return colonyAddress Address of the newly created colony
  function createColony(address _tokenAddress) public returns (address);
```

Reply:
That's a mistake in the natspec really. I will fix